src/utils.js

```diff
@@ -29,11 +29,11 @@ export const OnboardingAPIs = {
 };
 
 export const CapabilityAPI = {
-	getHelpCenterCapability: () =>
-		apiFetch( {
-			path: base + '/capability',
-			method: 'GET',
-		} ),
+	getHelpCenterCapability: () => {
+		return (
+			window.NewfoldRuntime?.capabilities?.canAccessHelpCenter || false
+		);
+	},
 	getBrand: () =>
 		apiFetch( {
 			path: base + '/capability/brand',
```

Comment:
@abhijitb - Let's also replace the API call to `capability/brand` and use the `NewfoldRuntime.plugin.id` instead. This API is also being called everytime the capability one is, so it has the same issue.